src/Compilers/CSharp/Portable/Compilation/CSharpSemanticModel.cs

```diff
@@ -467,8 +467,15 @@ private int CheckAndAdjustPositionForSpeculativeAttribute(int position)
 
         protected override IOperation GetOperationCore(SyntaxNode node, CancellationToken cancellationToken)
         {
+            if (!Root.FullSpan.Contains(node.FullSpan))
+            {
+                // given node must be sub node of this root.
+                return null;
```

Comment:
actually, looks like Root only exist in each language's semantic SemanticModel not in the common one. should I expose Root (internal) from SemanticModel?